Sources/SWBAndroidPlatform/CMakeLists.txt

```diff
@@ -37,3 +37,8 @@ target_link_libraries(SWBAndroidPlatform PUBLIC
   SWBUtil)
 target_sources(SWBAndroidPlatform PRIVATE
   "${CMAKE_CURRENT_BINARY_DIR}/resource_bundle_accessor.swift")
+
+target_include_directories(SWBAndroidPlatform PUBLIC
```

Comment:
I'm not confident `target_include_directories` is the right way to be propagating module search paths to clients

Reply:
When you say clients, are you referring to external projects, or to other libraries within this project?

If swift-build is intended to be linked against by other projects, you'll want to be careful about exposing absolute paths into your build directory like this, unless the project is only ever intended to be used from the build directory.

```
target_include_directories(SWBAndroidPlatform PUBLIC
  $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
  $<INSTALL_INTERFACE: -- wherever you're installing these files to -->)
```

https://cmake.org/cmake/help/latest/command/target_include_directories.html#creating-relocatable-packages